src/bidiMapper/modules/cdp/CdpTarget.ts

```diff
@@ -254,15 +254,11 @@ export class CdpTarget {
         this.toggleDeviceAccessIfNeeded(),
       ]);
     } catch (error: any) {
-      this.#logger?.(LogType.debugError, 'Failed to unblock target', error);
-      // The target might have been closed before the initialization finished.
-      if (!this.#cdpClient.isCloseError(error)) {
-        this.#unblocked.resolve({
-          kind: 'error',
-          error,
-        });
-        return;
-      }
+      this.#logger?.(
```

Comment:
Let move to Promise.allSettled and throw after so that we know the other thing are already set up.

Reply:
we do not need to throw here. Like 50% of thsese calls throw  see https://github.com/GoogleChromeLabs/chromium-bidi/issues/3702